docs/querying/functions.md

```diff
@@ -73,6 +73,11 @@ For each input time series, `changes(v range-vector)` returns the number of
 times its value has changed within the provided time range as an instant
 vector.
 
+## `clamp()`
+
+`clamp(v instant-vector, min scalar, max scalar)`
+clamps the sample values of all elements in `v` to have a lower limit of `min` and an upper limit of `max`.
```

Comment:
```suggestion
clamps the sample values of all elements in `v` to have a lower limit of `min` and an upper limit of `max`.

Special cases:
- Return an empty vector if `min > max`.
- Return `NaN` if `min` or `max` is `NaN`.
```

Reply:
Added these special cases in docs and test cases.